pkg/util/sqlite/sqlite_nocgo.go

```diff
@@ -0,0 +1,17 @@
+//go:build !cgo
+
+package sqlite
+
+import "modernc.org/sqlite"
+
+type Driver = sqlite.Driver
```

Comment:
Can we add a comment on this that it is expected that it won't work well for now? :)